addon/components/flexberry-map.js

```diff
@@ -185,6 +185,24 @@ let FlexberryMapComponent = Ember.Component.extend(
     layers: null,
 
     /**
+      Layer links object thats use for query data on map load
+
+      @property layerLinks
+      @type Ember.Array
+      @default null
+    */
+    layerLinks: null,
+
+    /**
+      JSON object with query parameters
+
+      @property queryFilter
+      @type object
+      @default null
+     */
+    queryFilter: null,
```

Comment:
Maby it is better to pass queryFilter into 'flexberry-map:query' event handlers, and not store it as property of the layer instance.